requirements/datatype_image_extras.txt

```diff
@@ -8,7 +8,6 @@ effdet
 albumentations
 learn2learn
 structlog==21.1.0   # remove when baal resolved its dependency.
```

Comment:
Can we remove this too then?

Reply:
Attempted the remove in the new commits. Waiting for the CI to turn 🟢! :)